PREREQUISITES.md

```diff
@@ -174,12 +174,18 @@ install the Intel SGX driver for both standalone and docker builds.
 You need to install the Intel SGX driver whether you build TCF standalone
```

Comment:
Could you please segregate these changes into 2 commits?

Reply:
I separated the commits into 2 sets:
-  Installation documentation changes in `PREREQUISITES.md` and `tcs_config.toml`
- Installation changes in `rebuild.sh`